examples/carousel/next.config.mjs

```diff
@@ -0,0 +1,23 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```

Comment:
something for later, I wonder if we just pull out frames-nextjs into it's own package to import these common settings

Reply:
> something for later, I wonder if we just pull out frames-nextjs into it's own package to import these common settings

The problem is, if we move them to a common package, we would have to publish this package for someone to only clone an example, I believe.